src/app/qgsdecorationlayoutextentdialog.cpp

```diff
@@ -42,6 +42,8 @@ QgsDecorationLayoutExtentDialog::QgsDecorationLayoutExtentDialog( QgsDecorationL
 
   updateGuiElements();
   connect( buttonBox->button( QDialogButtonBox::Apply ), &QAbstractButton::clicked, this, &QgsDecorationLayoutExtentDialog::apply );
+  connect( buttonBox, &QDialogButtonBox::helpRequested, this, &QgsDecorationLayoutExtentDialog::showHelp );
+// Pas écrit pareil ci-dessus
```

Comment:
Ah right... The ApplyRole isn't among and I now understand why my signal commands fail. Thanks.